flink-streaming-java/src/test/java/org/apache/flink/streaming/api/DataStreamTest.java

```diff
@@ -183,6 +183,14 @@ public Long map(Long value) throws Exception {
 
 		// verify self union
 		assertTrue(streamGraph.getStreamNode(selfUnion.getId()).getInEdges().size() == 2);
+		assertTrue(streamGraph.getUniqueEdgeMap().size() == 12);
+		int selfUnionCount = 0;
+		for (Integer value : streamGraph.getUniqueEdgeMap().values()) {
+			if (value == 2) {
```

Comment:
why constant value of `2`? where does it come from? Shouldn't it be something like `selfUnion.getId()`?